cip-constitution-tech/README.md

```diff
@@ -0,0 +1,255 @@
+
+## Motivation: why is this CIP necessary?
+
+CIP-1694 defines the on-chain anchor mechanism used to link the off-chain constitution document to on-chain actions.
+This mechanism was chosen due to its simplicity and cost effectiveness, moving the potentially large Cardano constitution off chain, referencing it via a hash digest and URI.
+This is the extent to which CIP-1694 outlines the constitution, it does provide suggestions around hashing algorithm, off-chain storage location, rich text styling.
+
+By formalizing the form of the constitution and it's iterations, we aim to promote it's longevity and accessibility.
+This is essential for the governance model
+
+Like the Cardano constitution this standard impacts every ada holder.
+
+### Safety
+
+Without describing best practices for the form and handling of the constitution, we risk the constitution document being stored in an insecure manner.
+By storing the constitution on a decentralized platform, we can ensure it's immutability and permissionless access.
+This is a step to ensuing the longevity and accessibility of each constitution iteration.
+
+### Interoperability
+
+By defining a file extension and formatting rules for the constitution we ensure that tooling reading and writing the constitution will be interoperable.
+Furthermore we aim to make roles constitution iteration comparison tools easy, by minimizing formatting and style changes between iterations.
+This will reduce compatibility issues between tools, promoting the accessibility of the constitution.
+
+### Usability
+
+Rich text formatting greatly enhances the readability of text, especially in large complex documents.
+Without the ability to format text, it could easily become cumbersome to read, negatively effecting the accessability of the Cardano constitution.
+
+## Specification
+
+The following specification should be applied to all constitution iterations.
+This standard could be augmented in the future via a separate CIP which aims to replace this one.
+
+### File Type
+// todo - any suggestions?
+
+### Hashing
+
+When supplying a constitution hash digest to chain, the algorithm used MUST be Blake2b-256.
+Before creating a hash digest the constitution plain text must be in its raw un-rendered format.
+
+- add a way to change algorithm?
+
+### Storage
+
+The each ratified constitution MUST be stored, immutably on a distributed storage mechanism (such as IPFS).
+Where backups can be easily made in a permission less manner by interested parties.
+Must be easily accessible, normal tooling can support it.
+Authors SHOULD NOT specify the use of centralized gateways to access the constitution plain text.
+
+### Rich Text Formatting
+
+The constitution text MAY include a subset of markdown text styling.
+Tooling rendering the constitution for users, should recognize these and render them faithfully.
+
+#### Headers
```

Comment:
If this is to specify the markdown subset, it probably needs a quick intro: "this is the acceptable specified subset of markdown text styling"

Reply:
Good catch

Reply:
fixed the wording